deltacat/tests/test_utils/storage.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Dict
+
+from deltacat.aws.redshift import Manifest, ManifestMeta
+from deltacat.storage import Partition, DeltaType, DeltaLocator, Delta
+from deltacat.utils.common import current_time_ms
+
+
+def create_empty_delta(
```

Comment:
Minor, this can actually belong to [local_deltacat_storage/__init__.py](https://github.com/ray-project/deltacat/pull/225/files#diff-f03e453d0c87446afab06f5045c36f0a16c23fafb51004f652f266e63059f1f9). I had deleted that earlier comment.

Reply:
Yeah, I noticed that as well, but I figured separating out test helper methods from the local storage impl. would be easier to read and maintain in the future.